java/demo/src/main/java/com/google/phonenumbers/demo/helper/LibPhoneNumberTemplate.java

```diff
@@ -0,0 +1,32 @@
+package com.google.phonenumbers.demo.helper;
+
+import com.google.template.soy.SoyFileSet;
+import com.google.template.soy.data.SoyTemplate;
+import com.google.template.soy.tofu.SoyTofu;
+import java.net.URL;
+
+public class LibPhoneNumberTemplate {
+  private final String template;
+  private final String namespace;
+  private final SoyTemplate soyTemplate;
+
+  public LibPhoneNumberTemplate(String template, String namespace, SoyTemplate soyTemplate) {
+    this.template = template;
+    this.namespace = namespace;
+    this.soyTemplate = soyTemplate;
+  }
+
+  private static URL getResource(String template) {
```

Comment:
Doesn't need to be static anymore and doesn't need the param as it's available as a field

Reply:
Where you are right you are right ;)